PublishPackageManagementsUpdates.yml

```
@@ -56,6 +58,130 @@
         echo "Versions are the same. Skipping PyPI upload."
       fi
 
+  - job: CreateGitHubRelease
+  displayName: 'Create GitHub Release'
```

YAMLlint (1.37.1)

[error] duplication of key "displayName" in mapping

(key-duplicates)